Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```
@@ -637,3 +640,61 @@
+      mask:
+      - SmallImpassable
+      - MobImpassable
+      - VaultImpassable
+      - Impassable
+      layer:
+      - Opaque
+  - type: Appearance
+    visuals:
+    - type: DamageStateVisualizer
+      rotate: true
+      normal: onestar_boss  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)